### PR TITLE
Doc dontDisableStatic variable to configure stage

### PR DESCRIPTION
**Pull request by Havvy**

### CODE REVIEW

File: doc/stdenv.xml [hunk 1]
```diff
@@ -546,6 +546,15 @@ script) if it exists.</para>
     such.</para></footnote>.  If this is undesirable, set this
     variable to a non-empty value.</para></listitem>
   </varlistentry>
+  
+  <varlistentry>
+    <term><varname>dontDisableStatic</varname></term>
+    <listitem><para>By default, when the configure script has
+    <option>--enable-static</option>, the option
+    <option>--disable-static</option> is added to the configure flags.</para>
+    <para></para>If this is undesirable, set this variable to a non-empt
```
Comment (peti) [inline doc/stdenv.xml]: 'y' missing at the end of the line.

Comment (peti) [inline doc/stdenv.xml]: It's probably  best to recommend setting the variable to `true`, btw.

**Comment by Havvy** [comment]
I went with what the other boolean parameters said. Should I switch them to saying `true` too?

**Comment by Havvy** [comment]
Alright. The typo has been fixed. I'll open an issue on the `non-empty` thing too.

**Pull request merged**